lib/portlayer/storage/vsphere/image.go

```diff
@@ -222,7 +222,8 @@ func (v *ImageStore) WriteImage(ctx context.Context, parent *portlayer.Image, ID
 	// will be descended from this created and prepared fs.
 	if ID == portlayer.Scratch.ID {
 		// Create the disk
-		vmdisk, cerr := v.dm.CreateAndAttach(ctx, imageDiskDsURI, "", defaultDiskSize, os.O_RDWR)
+		log.Debugf("Default disk size reported as %d", portlayer.Config.ScratchSize)
+		vmdisk, cerr := v.dm.CreateAndAttach(ctx, imageDiskDsURI, "", int64(portlayer.Config.ScratchSize), os.O_RDWR)
```

Comment:
not anymore, thanks for noticing it, the output from the `units` conversion is an int64